app/config.js

```diff
@@ -23,7 +23,7 @@ window.OPENSHIFT_CONFIG = {
   },
   additionalServers: [{
     protocol: "https",
-    hostPort: "apiserver-svc-cat.10.0.0.25.xip.io:6443",
+    hostPort: "apiserver-svc-cat.10.0.0.25.xip.io",
```

Comment:
This value still won't work for most people unfortunately. If you're running cluster up locally on your laptop, it will be 127.0.0.1. I assume you're running cluster up in vagrant? It also relies on you having deployed the template, and eventually it won't be required.

We might just comment the stanza out in the default config.js and explain above how it's used. Perhaps default to mock data.